### PR TITLE
Mxnet-operator now support tvm auto-tuning

### PR DESCRIPTION
**Pull request by KingOnTheStar**
You can use mxnet-operator to deploy a tvm auto-tuning task. See example/mxnet-operator.v2/tune for details.
Mxnet-operator now will wrap all the environments variable into MX_CONFIG, user's code should request env from it if necessary.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/mxnet-operator/15)

### CODE REVIEW

File: examples/mxnet-operator.v2/tune/mx_job_tune_gpu.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+          - name: mxnet
+            image: kingonthestar/auto-tuning:2.0
+            command: ["python3"]
+            args: ["/home/start-job.py"]
+    TunerServer:
+      label: gpu1
```
Comment (gaocegege) [inline examples/mxnet-operator.v2/tune/mx_job_tune_gpu.yaml]: Why do we need such a label?

**Comment by gaocegege** [comment]
/ok-to-test

Thanks for your contribution!

File: examples/mxnet-operator.v2/tune/mx_job_tune_gpu.yaml (hunk 1 above)
Comment (suleisl2000) [reply examples/mxnet-operator.v2/tune/mx_job_tune_gpu.yaml]: We need use the type of GPU card as label here. Furthermore, we have to update document to guide user how to use the feature. For example, what the label is used for, how user programming their tuning code, how to configure scheduling policy for the feature and so on.

File: examples/mxnet-operator.v2/tune/mx_job_tune_gpu.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      replicas: 1
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+          - name: mxnet
+            image: kingonthestar/auto-tuning:2.0
```
Comment (jzp1025) [inline examples/mxnet-operator.v2/tune/mx_job_tune_gpu.yaml]: the image name should be more official , we can use a specific account to store images. And the user can choose what they need from that account

**Comment by KingOnTheStar** [comment]
@jzp1025 I have pushed docker image mxjob/auto-tuning:gpu, and correct the yaml file.

**Pull request merged**